myning/chapters/blacksmith.py

```diff
@@ -80,6 +80,16 @@ def enter(self):
             ],
         )
 
+    def confirm_multi_buy(self, items: list[Item]):
```

Comment:
I'm confused. This doesn't exist yet, buy something is referencing it?

Reply:
This is a method on the `BaseStore` class that we're overriding here to add an extra check before calling the super's method. The regular store and the garden store also inherit from `BaseStore`. It's used here: https://github.com/TheRedPanda17/myning/blob/main/myning/chapters/base_store.py#L100

The `@override` decorator from this PEP https://peps.python.org/pep-0698/ will be helpful if they decide to include it in python in the future